.github/workflows/tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Tests
+
+on: [pull_request]
+
+jobs:
```

Comment:
Also needs a cache.

https://github.com/MerosCrypto/asmr/blob/master/.github/workflows/ci.yaml#L54-L84 for an example